File: fizzbuzz.py
# SOLVED: FizzBuzz Challenge.
for i in range(101):
    if i == 0:
        continue
    if i%3 == 0 and i%5 == 0:
        print(f"{i} | FizzBuzz")
    elif i%3 == 0:
        print(f"{i} | Fizz")
    elif i%5 == 0:
        print(f"{i} | Buzz")
    else:
        print(i)

print("==========================")

def fizzbuzz(range_num, first_num, second_num):
    for i in range(range_num):
        if i == 0:
            continue
        if i % first_num == 0 and i % second_num == 0:
            print(f"{i} | FizzBuzz")
        elif i % first_num == 0:
            print(f"{i} | Fizz")
        elif i % second_num == 0:
            print(f"{i} | Buzz")
        else:
            print(i)
            
fizzbuzz(200, 5, 8)

File: fizzbuzz.js
// SOLVED: FizzBuzz Challenge.
for (let i = 1; i <= 100; i++) {
    if (i % 3 === 0 && i % 5 === 0) {
        document.write("<p class='text-center'>" + i + " | FizzBuzz" + "</p>");
    } else if (i % 3 === 0) {
        document.write("<p class='text-center'>" + i + " | Fizz" + "</p>");
    } else if (i % 5 === 0) {
        document.write("<p class='text-center'>" + i + " | Buzz" + "</p>");
    } else {
        document.write("<p class='text-center'>" + i + "</p>");
    }
}


function fizzbuzz(range_num, first_num, second_num) {
    for (let i = 1; i <= range_num; i++) {
        if (i % first_num === 0 && i % second_num === 0) {
            document.write("<p class='text-center'>" + i + " | FizzBuzz" + "</p>");
        } else if (i % first_num === 0) {
            document.write("<p class='text-center'>" + i + " | Fizz" + "</p>");
        } else if (i % second_num === 0) {
            document.write("<p class='text-center'>" + i + " | Buzz" + "</p>");
        } else {
            document.write("<p class='text-center'>" + i + "</p>");
        }
    }
}

fizzbuzz(200, 5, 8);